src/pages/inputMethod.tsx

```diff
@@ -1,72 +1,108 @@
-                                    name="inputMethod"
-                                    type="radio"
-                                    value="manual"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="manual-entry">
-                                    Manual Fares Triangle input
-                                </label>
-                            </div>
-                            <div className="govuk-radios__item">
-                                <input
-                                    className="govuk-radios__input"
-                                    id="interactive-map"
-                                    name="inputMethod"
-                                    type="radio"
-                                    value="interactiveMap"
-                                    disabled
-                                    aria-disabled="true"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="interactive-map">
-                                    Interactive Map (Beta)
-                                </label>
+type InputMethodProps = {
+    errors: ErrorInfo[];
+};
+
+const InputMethod = ({ errors = [] }: InputMethodProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
+            <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
+                <form action="/api/inputMethod" method="post">
+                    <ErrorSummary errorHref="#radio-buttons" errors={errors} />
+                    <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                        <fieldset className="govuk-fieldset" aria-describedby="input-method-heading">
+                            <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
+                                <h1 id="input-method-heading" className="govuk-fieldset__heading">
+                                    Please select your preferred input method
+                                </h1>
+                                {errors.length > 0 && (
+                                    <span id="fareType-error" className="govuk-error-message error-message-padding">
+                                        <span>{errors[0].errorMessage}</span>
+                                    </span>
+                                )}
+                            </legend>
+                            <div className="govuk-radios" id="radio-buttons">
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className={`govuk-radios__input ${
+                                            errors.length > 0 ? 'govuk-input--error' : ''
+                                        } `}
```

Comment:
Same here as the faretype and the howManyStages pages

Reply:
Done